src/main/java/io/github/lofienjoyer/nubladatowns/town/TownManager.java

```diff
@@ -66,6 +67,7 @@ public void loadData(YamlConfiguration dataConfig) {
             town.setRgbColor(section.getInt("color"));
             town.setSpawn(section.getLocation("spawn"));
             town.setOpen(section.getBoolean("open", true));
+            town.setPower(section.getInt("power"));
```

Comment:
Add a 0 as a second argument in the getInt method as default value

Reply:
Fixed in commit 8bc11b3